cli/cmd/sync_v0.go

```diff
@@ -84,6 +85,12 @@ func syncConnectionV0_2(ctx context.Context, sourceClient *managedsource.Client,
 		return err
 	}
 	writeClients := make([]destination.Destination_Write2Client, len(destinationsPbClients))
+	defer func() {
+		for i := range writeClients {
+			_ = writeClients[i].CloseSend()
```

Comment:
can you check err and print it to the log so we see that we are not hiding the same error that we were fixing?

Reply:
done in 9f1eed6ecd56ca45c169ec153c9059eee5a5aed0